apps/docs/pages/guides/realtime/rate-limits.mdx

```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
 
 Rate limits are configurable per project and our cluster supports millions of concurrent connections. [Contact support](https://app.supabase.com/support/new) if these limits are causing issues.
 
-## Limits by Plan
+## Rate Limits by Plan
 
-Limits are set on the Free and Pro plans accordingly. See our [pricing page](https://supabase.com/pricing) for more information.
+Rate limits are different than usage limits. The usage limits for Free and Pro projects can be found on the See our [pricing page](https://supabase.com/pricing).
 
-Enterprise plans are billed on a usage basis. We still employ limiting on Enterprise plans. If you're on an Enterprise plan simply contact support and we'll increase your limits as needed.
+Rate limits protect our infrasructure. You can increase your rate limits by upgrading your plan and removing your spend cap.
 
-Enterprise plan limits start at:
+If you need your rate limits increased further please [contact support](https://app.supabase.com/support/new).
+
+We have different rate limits per plan:
+
+### Free
+
+- 200 concurrent clients
+- 100 messages per second
+- 100 Channel joins per second
+
+### Pro
 
 - 500 concurrent clients
+- 500 messages per second
+- 500 Channel joins per second
+
+### Pro with no spend cap
+
+Starts at:
+
+- 5,000 concurrent clients
+- 1,000 messages per second
+- 1,000 Channel joins per second
+
+### Enterprise
+
+Starts at:
+
+- 5,000 concurrent clients
 - 1,000 messages per second
-- 500 concurrent Channels
+- 1,000 Channel joins per second
 
 ## System Limits
 
 The following limits apply to all projects:
 
-- 500 Channel joins per second
 - 100 Channels per connected client
```

Comment:
@chasers these are both defaults so if you take out one you should take out the other. both of these can be customized so they don't technically apply to all projects and they currently don't apply to all projects.